### PR TITLE
Add writegood maker for text, markdown, pandoc

### PR DESCRIPTION
**Pull request by blueyed**
Closes https://github.com/neomake/neomake/pull/773.

TODO:
 - [ ] Needs an unreleased version?!
https://github.com/btford/write-good/commit/a4e52ae33cff729e78e8f502a4ce37e7f7aa9887#commitcomment-19895355
 - [ ] does not have all errors/warning on a single line (https://github.com/btford/write-good/commit/a4e52ae33cff729e78e8f502a4ce37e7f7aa9887#commitcomment-19895374).

### CODE REVIEW

**Comment by blueyed** [comment]
@Alok @jondot
Please check if this PR works for you (see https://help.github.com/articles/checking-out-pull-requests-locally/).

**Comment by blueyed** [comment]
Closing this for now - can be re-opened when questions get answered, and I do not have the feeling to maintain this myself.

**Comment by bitboxer** [comment]
@blueyed I would love to investigate into this in the next couple of days. Would that be okay? Would also maintain this in here if needed.

**Comment by blueyed** [comment]
@bitboxer 
Sure! :)
I suggest picking up this PR and create a new one based on it then.

**Comment by bitboxer** [comment]
Perfect. Will do. First thing I did was to ask when the next release of write-good will happen :wink:

**Comment by bitboxer** [comment]
write-good just released 0.10.0 . Will create a new PR with this in the next week.

**Comment by bitboxer** [comment]
Sadly the `--parse` option has still the multi line issue. Opened [a ticket](https://github.com/btford/write-good/issues/59) for this.

**Comment by bitboxer** [comment]
And created a PR to fix that problem :wink:

**Comment by bitboxer** [comment]
I only have this one question, besides that I am okay with merging this.

File: autoload/neomake/makers/ft/markdown.vim [hunk 1]
```diff
@@ -1,6 +1,6 @@
 function! neomake#makers#ft#markdown#EnabledMakers() abort
     let makers = executable('mdl') ? ['mdl'] : ['markdownlint']
-    return makers + ['proselint']
+    return makers + ['writegood'] + neomake#makers#ft#text#EnabledMakers()
```
Comment (blueyed) [inline autoload/neomake/makers/ft/markdown.vim]: Good catch.  Should have been both proselint and writegood.
Amended and squashed, please review again.

**Comment by blueyed** [comment]
Thanks for the review! :)

**Pull request merged**